package.json

```
@@ -64,6 +66,7 @@
     "@typescript-eslint/parser": "6.4.1",
     "babel-loader": "9.1.3",
     "css-loader": "6.8.1",
+    "dotenv": "^16.3.1",
```

давай версию зафиксируем как у всех остальных модулей
```suggestion
    "dotenv": "16.3.1",
```

---

👌